CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 1.12.0-rc1 / 2021-02-03
+
+* [BUGFIX] Add signature verification for updater
+* [BUGFIX] Allow using tilde
```

Comment:
We might want to have clearer bugfixes in the future. I guess it's "in the config file", but anyway, not important

Reply:
Yes, these should be more specific ...